src/main/java/io/github/thebusybiscuit/slimefun4/core/commands/subcommands/VersionsCommand.java

```diff
@@ -28,35 +32,75 @@ public void onExecute(@Nonnull CommandSender sender, @Nonnull String[] args) {
             // so we will just fix this inconsistency for them :)
             String serverSoftware = PaperLib.isSpigot() && !PaperLib.isPaper() ? "Spigot" : Bukkit.getName();
 
-            sender.sendMessage(ChatColor.GRAY + "This Server uses the following setup of Slimefun:");
-            sender.sendMessage(ChatColors.color("&a" + serverSoftware + " &2" + Bukkit.getVersion()));
-            sender.sendMessage(ChatColors.color("&aSlimefun &2v" + SlimefunPlugin.getVersion()));
+            ComponentBuilder builder = new ComponentBuilder();
+            builder.append("This Server uses the following setup of Slimefun:\n").color(ChatColor.GRAY)
+                .append(serverSoftware).color(ChatColor.GREEN).append(" " + Bukkit.getVersion() + '\n').color(ChatColor.DARK_GREEN)
+                .append("Slimefun").color(ChatColor.GREEN).append(" v" + SlimefunPlugin.getVersion() + '\n').color(ChatColor.DARK_GREEN);
 
             if (SlimefunPlugin.getMetricsService().getVersion() != null) {
-                sender.sendMessage(ChatColors.color("&aMetrics build: &2#" + SlimefunPlugin.getMetricsService().getVersion()));
+                builder.append("Metrics build: ").color(ChatColor.GREEN)
+                    .append("#" + SlimefunPlugin.getMetricsService().getVersion() + '\n').color(ChatColor.DARK_GREEN);
             }
 
+            addJavaVersion(builder);
+
             if (SlimefunPlugin.getRegistry().isBackwardsCompatible()) {
-                sender.sendMessage(ChatColor.RED + "Backwards compatibility enabled!");
+                builder.append("Backwards compatibility enabled!\n").color(ChatColor.RED);
```

Comment:
Should probably also add a tooltip here.

Reply:
yeah i was going to but really wasn't sure how to write it.

"If you don't have items older than Summer 2019 you should disable this to increase performance"
maybe?

Reply:
"Backwards compatibility has a negative impact on performance!
We recommend you to disable this setting unless your Server still
has legacy Slimefun items (from before summer 2019) in circulation."